content/docs/user-guide/deploying/kubernetes.md

```diff
@@ -12,6 +21,344 @@ pip install mlem[kubernetes]
+configured in the inbound rules of the node's security group.
+
+This can be seen as the last rule being added below:
+
+![alt text](/img/inbound.png)
+
+</admon>
+
+## Swapping the model in deployment
+
+If you want to change the model that is currently under deployment, simply run
+
+```cli
+$ mlem deploy run service_name --model other-model
+```
+
+This will build a new docker image corresponding to the `other-model` and will
+terminate the existing pod and create a new one, thereby replacing it, without
+downtime.
+
+This can be seen below:
+
+### Checking the docker images
+
+```
+REPOSITORY                                                TAG                                IMAGE ID       CREATED             SIZE
+342840881361.dkr.ecr.us-east-1.amazonaws.com/classifier   d57e4cacec82ebd72572d434ec148f1d   9bacd4cd9cc0   11 minutes ago      2.66GB
+342840881361.dkr.ecr.us-east-1.amazonaws.com/classifier   4ee45dc33804b58ee2c7f2f6be447cda   26cb86b55bc4   About an hour ago   778MB
+...
+```
+
+Notice how a new docker image with the tag `d57e4cacec82ebd72572d434ec148f1d` is
+built.
+
+### Checking the deployment process
+
+```
+⏳️ Loading deployment from service_name.mlem
+⏳️ Loading model from other-model.mlem
+🛠 Creating docker image classifier
+  🛠 Building MLEM wheel file...
+  💼 Adding model files...
+  🛠 Generating dockerfile...
+  💼 Adding sources...
+  💼 Generating requirements file...
+  🛠 Building docker image 342840881361.dkr.ecr.us-east-1.amazonaws.com/classifier:d57e4cacec82ebd72572d434ec148f1d...
+  🗝 Logged in to remote registry at host 342840881361.dkr.ecr.us-east-1.amazonaws.com
+  ✅  Built docker image 342840881361.dkr.ecr.us-east-1.amazonaws.com/classifier:d57e4cacec82ebd72572d434ec148f1d
+  🔼 Pushing image 342840881361.dkr.ecr.us-east-1.amazonaws.com/classifier:d57e4cacec82ebd72572d434ec148f1d to 342840881361.dkr.ecr.us-east-1.amazonaws.com
+  ✅  Pushed image 342840881361.dkr.ecr.us-east-1.amazonaws.com/classifier:d57e4cacec82ebd72572d434ec148f1d to 342840881361.dkr.ecr.us-east-1.amazonaws.com
+✅  Deployment classifier is up in mlem namespace
+```
+
+Here, an existing deployment i.e. `service_name` is used but with a newer model.
+Hence, details of registry need not be passed again. The contents of
+`service_name` can be checked by inspecting the `service_name.mlem` file.
+
+### Checking the kubernetes resources
+
+We can see the existing pod being terminated and the new one running in its
+place below:
+
+```
+NAMESPACE     NAME                          READY   STATUS        RESTARTS      AGE
+kube-system   aws-node-pr8cn                1/1     Running       0             90m
+...
+kube-system   kube-proxy-dfxsv              1/1     Running       0             90m
+mlem          classifier-66b9588df5-wmc2v   1/1     Running       0             99s
+mlem          classifier-687655f977-bm4w8   1/1     Terminating   0             60m
+```
+
 ## Examples
```

Comment:
Do we need examples here? Maybe running some special commands like `mlem deploy X`, if you didn't cover them above? Or maybe something else? We may need this if we move "Case study" somewhere though. Otherwise there is a lot of info above, maybe that's enough for now.

Reply:
Yes, I renamed `Case Study` to `Example`, so that solves it.